src/main/kotlin/task11/ktor/task02/DbUtils.kt

```diff
@@ -11,18 +11,13 @@ val dataSource: DataSource = HikariDataSource(HikariConfig().apply {
     jdbcUrl = "jdbc:h2:mem:testdb;MODE=PostgreSQL;DB_CLOSE_DELAY=-1"
 })
 
-fun prepopulateDatabase(dataSource: DataSource) {
+fun createDbTables(dataSource: DataSource) {
     @Language("H2")
     val sqlsetup: String = """
     CREATE TABLE cocktail(id VARCHAR(36) NOT NULL, name VARCHAR(100) NOT NULL);
     CREATE TABLE ingredient(id VARCHAR(36) NOT NULL, cocktail_id VARCHAR(36) NOT NULL, name VARCHAR(100) NOT NULL);
-
-    INSERT INTO cocktail VALUES ('9b59090e-9968-4e43-9675-859f6cc64fda', 'Margarita');
```

Comment:
Kanskje beholde denne og neste linje så man kan se på SQL syntax hvis aldri har brukt det før

Reply:
margarita med bare tequila ble litt for hardcore for min del, så la med de andre ingrediensene også. ;)